client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -48,8 +49,9 @@ const SiteCreationStep: Step = function SiteCreationStep( { navigation, flow } )
 		siteVisibility = Site.Visibility.PublicNotIndexed;
 	}
 
-	// Ecommerce flow defaults to Private
-	if ( flow === ECOMMERCE_FLOW ) {
+	// Certain flows should default to private.
+	const privateFlows = [ ECOMMERCE_FLOW, WOOEXPRESS_FLOW ];
+	if ( privateFlows.includes( flow ) ) {
```

Comment:
Actually - it looks like there is a type issue here as `flow` may be null. My suggestion may be enough to get that resolved, but we also check for `if ( flow && privateFlows.includes( flow ) )`.
```suggestion
	if ( privateFlows.includes( flow || '' ) ) {
```